src/Songs.tsx

```diff
@@ -1,5 +1,4 @@
 /* Data structures to support handling songs */
-import '@capacitor-community/http';
```

Comment:
@brotatotes it looks like the automated build checker works haha, and it doesn't like this line. The app still ran fine on my computer locally with it, but fails the CI build. Lets try to fix the build, but if this line does need to be here like this then I can go tinker with the build stuff somemore later. 

This was the build failure log.
https://app.circleci.com/pipelines/github/Church-Life-Apps/Songs/39/workflows/c8c0e23f-e58a-4813-8128-008e88e5b9e5/jobs/51

Reply:
Sound good. I think that might not be needed.